swarm/pss/notify/notify_test.go

```diff
@@ -145,17 +145,24 @@ func TestStart(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	copyOfUpdateMsg := make([]byte, len(updateMsg))
```

Comment:
This copy is future proof, but I was thinking about just inlining `byte[]{}` in the lambda. What do you think?

Reply:
I think that copy is ok, maybe a comment why do we need it, but not required.